libnmstate/ifaces/bond.py

```diff
@@ -191,6 +192,30 @@ def remove_port(self, port_name):
         ]
         self.sort_port()
 
+    def match_ignore_bond_options(self, current):
+        self_state = self.state_for_verify()
+        self_bond_options = self_state.get(Bond.CONFIG_SUBTREE, {}).pop(
+            Bond.OPTIONS_SUBTREE, {}
+        )
+        current_state = current.state_for_verify()
+        current_bond_options = current_state.get(Bond.CONFIG_SUBTREE, {}).pop(
+            Bond.OPTIONS_SUBTREE, {}
+        )
+        for key, value in self_bond_options.items():
+            current_value = current_bond_options.get(key)
+            if current_value != value:
+                if current_value is None:
+                    logging.warning(
+                        f"Desire iface {self.name} bond option {key}={value} "
+                        f"is invalid in kernel for bond mode {self.bond_mode}"
+                    )
+                else:
+                    logging.warning(
+                        f"Desire iface {self.name} bond option {key}={value} "
+                        f"does not match with kernel value: {current_value}"
+                    )
```

Comment:
Is weird to me that something that is not valid is raising a warning instead of failing.. I am approving this but I would like to think on a better solution :-)

Reply:
Old nmstate did not raise exception on invalid bond options. So show now even it's wrong.

And oVirt refused to use `verify_change=False` for bond options due to their own limitation.